kibana-reports/public/components/report_definitions/report_settings/report_settings.tsx

```diff
@@ -82,7 +82,24 @@ function TimeRangeSelect(props) {
   const [refreshInterval, setRefreshInterval] = useState();
 
   useEffect(() => {
-    parseTimeRange(start, end, reportDefinitionRequest);
+    // if we are coming from the in-context menu
+    if (window.location.href.indexOf("?") > -1) {
+      const url = window.location.href;
+      const reportSourceInfo = url.split("?");
+      const timeFrom = reportSourceInfo[2].substring(
+        reportSourceInfo[2].indexOf("=") + 1,
+        reportSourceInfo[2].length
+      );
```

Comment:
could we refactor to a reusable parse method

Reply:
Yes, will change